include/taskolib/Sequence.h

```diff
@@ -279,20 +280,18 @@ class Sequence
     void execute(Context& context, CommChannel* comm_channel);
 
     /**
-     * Get the step setup script.
+     * Return an optional Error object explaining why the sequence stopped prematurely.
```

Comment:
I think you did not only exchange `get_error_message()` for `get_error()`, but ALSO changed the position where it occurs in the source file. That makes very ugly diffs. I tend to put these changes into different commits, to make review easier, like 'reorder functions, no functional change'.

Reply:
Well, I added a new function and then removed the old one. Agree, this makes the diff ugly. Actually the alphabetic ordering of the functions is still broken, this is probably where this came from. I might add a small commit to rectify that.

Reply:
Fixed.